.travis.yml

```diff
@@ -21,22 +22,23 @@ before_install:
 install:
   - pip freeze -l
   # Preinstalled travis pytest is not tracked by us
-  - pip uninstall pytest -y
+  # Preinstalled travis pbr is not tracked by us
```

Comment:
Thanks for pointing out the `pip` idiosyncrasies, will update the install commands with the `-U` option and remove the explicit removals. Still keeping the `pip freeze -l` commands since we have several pip installs from various requirements files and I want that info in there for debugging purposes.